common/changes/master_2017-04-12-17-48.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fix issue #1274",
```

Comment:
This should be something like `DatePicker: Persist selected date across rerenders`. Something that describes the component and change without referencing the Issue#, as it will be included in release notes.